doc/source/user_guide/troubleshoot.rst

```diff
@@ -139,6 +139,8 @@ Or, you can specify the directory using the ``-dir`` flag:
 
 If this command doesn't launch MAPDL, look at the command output:
 
+.. vale off
```

Comment:
@germa89 You might want to talk to @RobPasMue on this. My understanding was that Vale wouldn't check what appeared in code blocks.

Reply:
We will investigate this in another PR. So far this approach works. 

I have also open an issue here:

https://github.com/errata-ai/vale/issues/619